CHANGELOG.md

```diff
@@ -68,6 +68,8 @@ Main (unreleased)
 - Add `agent_wal_out_of_order_samples_total` metric to track samples received
   out of order. (@rfratto)
 
+- Add CLI flag `--enable-go-profiling` to grafana-agent-flow to conditionally enable `/debug/pprof` endpoints (@jkroepke)
```

Comment:
Can you document the new flag at https://github.com/grafana/agent/blob/main/docs/sources/flow/reference/cli/run.md?

Reply:
Done